tidy3d/components/geometry/base.py

```diff
@@ -1811,7 +1880,7 @@ def surfaces_with_exclusion(cls, size: Size, center: Coordinate, **kwargs):
         return surfaces
 
     @verify_packages_import(["trimesh"])
-    def intersections_tilted_plane(
+    def _do_intersections_tilted_plane(
```

Comment:
I think it should be fine, since the original `intersections_tilted_plane` is defined in the base class. The interface for these geometries is unchanged. Now, there is just a quick check to see if a simpler method can be used to compute the intersection, before falling back to the method involving generating a mesh.

Reply:
ok I see. thanks! should be fine then